authentication-typescript/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## v3.0.0 (Unreleased)
```

Comment:
You can set this to tomorrow in preparation for getting it released as part of this sprint

Reply:
Or you may want to play it safe and change the version to an alpha so we can test it all out before we make it official.